Makefile

```diff
@@ -157,7 +157,7 @@ performance_bench: performance server db_container
 
 .PHONY: performance_bench_report
 performance_bench_report: is_docker_running
-	docker container cp "$$(docker inspect --format="{{.Id}}" performance-performance-$(hash))":/core/simple-create_write_read.svg .
+	docker container cp "$$(docker inspect --format="{{.id}}" performance-performance-$(hash))":/core/simple-create_write_read.svg .
```

Comment:
seems like you have a rogue search and replace that's changed a few files you didn't intend to change.